mmdeploy/backend/rknn/onnx2rknn.py

```diff
@@ -40,6 +42,14 @@ def onnx2rknn(onnx_model: str,
     output_names = onnx_params.get('output_names', None)
     input_size_list = get_backend_config(deploy_cfg).get(
         'input_size_list', None)
+    # update norm value
```

Comment:
In my opinion, it's better to put `get_rknn_quantization` and `get_normalization` in `onnx2rknn.py` rather than `mmdeploy.utils`, cuz they are specially designed for rknn deployment.

Reply:
But we import these methods in codebases.

Reply:
Is it safe to call function in backends from codebase?